packages/core/admin/admin/src/content-manager/components/ConfigurationForm/Fields.tsx

```diff
@@ -58,7 +58,15 @@ const Fields = ({ attributes, fieldSizes, components, metadatas = {} }: FieldsPr
     const [name, { visible, ...field }] = current;
 
     if (!existingFields.includes(name) && visible === true) {
-      acc.push({ ...field, label: field.label ?? name, name, size: fieldSizes[name] });
+      // @ts-expect-error – TODO: i'm not sure why it's convinced that attributes[name] can still be undefined after we've checked it.
```

Comment:
Can we make attributes not optional in
https://github.com/strapi/strapi/blob/93eda17d2ada24da9ee897406eb02bc37c1c7258/packages/core/admin/admin/src/content-manager/components/ConfigurationForm/EditFieldForm.tsx#L47

Reply:
oh god im dumb.